doc/HttpLuaModule.wiki

```diff
@@ -4698,6 +4699,56 @@ This feature was first introduced in the <code>v0.3.1rc22</code> release.
 
 See also [[#ngx.shared.DICT|ngx.shared.DICT]].
 
+== ngx.shared.DICT.cas ==
+'''syntax:''' ''success, err, forcible, current_value?, current_flags? = ngx.shared.DICT:cas(key, value, exptime, flags, old_value, old_flags?)''
+
+'''context:''' ''init_by_lua*, set_by_lua*, rewrite_by_lua*, access_by_lua*, content_by_lua*, header_filter_by_lua*, body_filter_by_lua*, log_by_lua*, ngx.timer.*''
+
+Just like the [[#ngx.shared.DICT.replace|replace]] method, but only stores the key-value pair into the dictionary [[#ngx.shared.DICT|ngx.shared.DICT]] if the <code>old_value</code> argument and <code>old_flags</code> argument ''does'' match the value and flags in the dictionary [[#ngx.shared.DICT|ngx.shared.DICT]] (or one of them).
```

Comment:
I think it's better to replace "if" with "if and only if".

The "or one of them" is confusing. In fact, when both arguments are specified (non-nil), both of the values must match instead of just one of them matches. I think you really mean that the arguments are optional and can take `nil` values, in which case the checks are omitted. Let's be explicit rather than confusing :)